### DISCUSSION TITLE
How to document nations and islands in or bordering the Caribbean Sea

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1488] [HOW TO...]
DISCUSSION POINTS: 

esites Topic 61: Documenting nations and islands in or bordering the Caribbean Sea

What’s the full hierarchy?

**Comment by rondlg** [comment]
# UPDATED DECISION BELOW: Island group hierarchy: **West Indies**

- LocIslandGrouping = West Indies
- - Antilles
- - - Lesser Antilles
- - - - Leeward Antilles
- - - - - LocIslandName = Aruba
- - - - - LocIslandName = Bonaire
- - - - - LocIslandName = Curaçao
- - - - - Federal Dependencies of Venezuela
- - - - - - LocIslandName = Aves Island; Los Monjes Archipelago; La Tortuga Island;  La Sola Island;  Los Testigos Islands; Los Frailes Islands; Patos Island; Los Roques Archipelago
- - - - - State of Nueva Esparta (Venezuela)
- - - - - - LocIslandName = Margarita Island; Coche; Cubaga

- - - - Leeward Islands
- - - - - LocIslandName = Saint Martin
- - - - - LocIslandName = Saba
- - - - - LocIslandName = Sint Eustatius
- - - - - LocIslandName = Virgin Islands
- - - - - - British Virgin Islands
- - - - - - - LocIslandName = Anegada; Beef Island; Bellamy Cay; Buck Island; Carvel Rock; Cockroach Island; Cooper Island; Dead Chest Island; Diamond Cay; Dog Islands; East Seal Dog Island  Eustatia ;Fallen Jerusalem Island ;Frenchman's Cay; George Dog IslandGinger IslandGreat CamanoeGreat Dog IslandGreat Thatch; Great Tobago Island; Green Cay; Guana Island; The Indians; Jost Van Dyke; Little Camanoe; Little Cay; Little Jost Van Dyke; Little Thatch; Little Tobago; Little Wickams Cay; Marina Cay; Mosquito Island; Nanny Cay; Necker Island; Norman Island; Old Jerusalem Island; Oyster Rock; Pelican Island; Peter Island; Prickly Pear; Round Rock; Saba Rock ;Salt Island; Sandy Cay; Sandy Spit Scrub Island ;Tortola; Virgin Gorda ;West Dog Island; Wickams Cay

- - - - - - US Virgin Islands
- - - - - - - LocIslandName = St. Thomas; St. John; St. Croix

- - - - - - Spanish Virgin Islands
- - - - - - - LocIslandName = Culebra; Vieques; Icacos Island; Cayo Lobo; Cayo Diablo; Palomino Island; Palominito Island; Isla de Ramos; Isla Pineiro; Cayo Lobo; Cayo Luis Peña

- - - - - LocIslandName = Dominica
- - - - - LocIslandName = Guadeloupe
- - - - - LocIslandName = Montserrat
- - - - - LocIslandName = Antigua
- - - - - LocIslandName = Barbuda
- - - - - LocIslandName = St. Kitts
- - - - - LocIslandName = Nevis
- - - - - LocIslandName = Anguilla

- - - - Windward Islands
- - - - - Grenada
- - - - - -  LocIslandName = Carriacou; Petite Martinique
- - - - - LocIslandName = Martinique
- - - - - LocIslandName = Barbados
- - - - - LocIslandName = St. Lucia
- - - - - LocIslandName = St. Vincent and the Grenadines

- - - Greater Antilles
- - - - Cuba
- - - - - Cayo Las Brujas
- - - - - - Canarreos Archipelago
- - - - - - - LocIslandName = Cayo Largo del Sur; Ernst Thälmann Island; Isla de la Juventud
- - - - - - Colorados Archipelago
- - - - - - - - LocIslandName = Banco Sancho Pardo; Cayo Alacranes; Cayo Buenavista; Cayo Ines de Soto; Cayo Jutías; Cayo Levisa; Cayo Punta Arenas; Cayo Rapado Grande; Cayo Santa María; Cayos Punta Tabaco
- - - - - - Jardines de la Reina
- - - - - - - - LocIslandName = Cayo Alcatraz; Cayo Algodon Grande; Cayo Anclitas; Cayo Breton; Cayo Caballones; Cayo Cabeza Del Este; Cayo Caguamas; Cayo De Las Doce Leguas; Cayos Ana Maria; Cayos Cinco Balas; Cayos Granada; Cayos Media Luna; Cayos Mordazo; Cayos Pingues
- - - - - - Jardines del Rey
- - - - - - - - LocIslandName = Cayo Coco; Cayo Esquivel; Cayo Guajaba; Cayo Guillermo; Cayo Romano; Cayo Sabinal; Cayo Saetia
- - - - - - Sabana-Camagüey Archipelago
- - - - - - - - LocIslandName =Cayo Fragoso
- - - - LocIslandName = Hispaniola
- - - - LocIslandName =  Jamaica
- - - - LocIslandName = Puerto Rico

**Comment by rondlg** [comment]
# UPDATED DECISION SEE BELOW: Political hierarchy: **West Indies**

## LocContinent_tab = South America

**Sovereign Nations** 
- PolPD1 [LocCountry_tab] = Trinidad and Tobago

## LocContinent_tab = North America

**Sovereign Nations** 
- PolPD1 [LocCountry_tab] = Antigua and Barbuda	
- PolPD1 [LocCountry_tab] = Haiti
- PolPD1 [LocCountry_tab] = Bahamas	
- PolPD1 [LocCountry_tab] = Jamaica
- PolPD1 [LocCountry_tab] = Barbados	
- PolPD1 [LocCountry_tab] = Saint Lucia
- PolPD1 [LocCountry_tab] = Cuba	
- - PolPD2 [LocProvinceStateTerritory_tab] 
- - - Rank=Province: Pinar del Río; Artemisa; La Habana; Mayabeque; Matanzas; Cienfuegos; Villa Clara; Sancti Spíritus; Ciego de Ávila; Camagüey; Las Tunas; Granma; Holguín; Santiago de Cuba; Guantánamo
- - PolPD2 [LocProvinceStateTerritory_tab] 
- - - Rank=Special Municipality: Isla de la Juventud 
- PolPD1 [LocCountry_tab] = Saint Kitts and Nevis
- PolPD1 [LocCountry_tab] = Dominica	
- PolPD1 [LocCountry_tab] = Saint Vincent and the Grenadines
- PolPD1 [LocCountry_tab] = Dominican Republic	
- PolPD1 [LocCountry_tab] = Grenada

**Dependant Territories**
SitSovereigntyRef = United Kingdom
- PolPD1 [LocCountry_tab] = Anguilla; 
- PolPD1 [LocCountry_tab] = Virgin Islands (British)
- PolPD1 [LocCountry_tab] = Cayman Islands
- PolPD1 [LocCountry_tab] = Montserrat
- PolPD1 [LocCountry_tab] = Turks and Caicos Islands
- PolPD1 [LocCountry_tab] = Aguilla

SitSovereigntyRef = Netherlands
- PolPD1 [LocCountry_tab] = Aruba
- PolPD1 [LocCountry_tab] =  Curaçao
- PolPD1 [LocCountry_tab] =  Sint Maarten
- PolPD1 [LocCountry_tab] =  Bonaire
- PolPD1 [LocCountry_tab] =  Saba
- PolPD1 [LocCountry_tab] = Sint Eustatius

SitSovereigntyRef = United States
- PolPD1 [LocCountry_tab] = Puerto Rico

SitSovereigntyRef = Netherlands
- PolPD1 [LocCountry_tab] = Saint Barthélemy
- PolPD1 [LocCountry_tab] = Collectivity of Saint Martin
- PolPD1 [LocCountry_tab] = Guadeloupe
- PolPD1 [LocCountry_tab] = Martinique

SitSovereigntyRef = Venezuela
- PolPD1 [LocCountry_tab] = Nueva Esparta
- PolPD1 [LocCountry_tab] = Federal Dependencies of Venezuela

**Comment by rondlg** [comment]
SitJurisdiction field replaced by SitSovereigntyRef

**Comment by rondlg** [comment]
esites Working group #1084 - Updated to reflect inclusion of political and island parent attachments.

# DECISION: Island group hierarchy: **West Indies**

- Island Grouping = West Indies
- - Antilles
- - - Lesser Antilles
- - - - Leeward Antilles
- - - - - Island Name = Aruba
- - - - - Island Name = Bonaire
- - - - - Island Name = Curaçao
- - - - - Federal Dependencies of Venezuela
- - - - - - Island Name = Aves Island; Los Monjes Archipelago; La Tortuga Island;  La Sola Island;  Los Testigos Islands; Los Frailes Islands; Patos Island; Los Roques Archipelago
- - - - - State of Nueva Esparta (Venezuela)
- - - - - - LocIslandName = Margarita Island; Coche; Cubaga

- - - - Leeward Islands
- - - - - Island Name = Saint Martin
- - - - - Island Name = Saba
- - - - - Island Name = Sint Eustatius
- - - - - Island Name = Virgin Islands
- - - - - - British Virgin Islands
 - - - - - - Island Name = Anegada; Beef Island; Bellamy Cay; Buck Island; Carvel Rock; Cockroach Island; Cooper Island; Dead Chest Island; Diamond Cay; Dog Islands; East Seal Dog Island  Eustatia ;Fallen Jerusalem Island ;Frenchman's Cay; George Dog IslandGinger IslandGreat CamanoeGreat Dog IslandGreat Thatch; Great Tobago Island; Green Cay; Guana Island; The Indians; Jost Van Dyke; Little Camanoe; Little Cay; Little Jost Van Dyke; Little Thatch; Little Tobago; Little Wickams Cay; Marina Cay; Mosquito Island; Nanny Cay; Necker Island; Norman Island; Old Jerusalem Island; Oyster Rock; Pelican Island; Peter Island; Prickly Pear; Round Rock; Saba Rock ;Salt Island; Sandy Cay; Sandy Spit Scrub Island ;Tortola; Virgin Gorda ;West Dog Island; Wickams Cay

- - - - - - US Virgin Islands
- - - - - - - Island Name = St. Thomas; St. John; St. Croix

- - - - - - Spanish Virgin Islands
- - - - - - - Island Name = Culebra; Vieques; Icacos Island; Cayo Lobo; Cayo Diablo; Palomino Island; Palominito Island; Isla de Ramos; Isla Pineiro; Cayo Lobo; Cayo Luis Peña

- - - - - Island Name = Dominica
- - - - - Island Name = Guadeloupe
- - - - - Island Name = Montserrat
- - - - - Island Name = Antigua
- - - - - Island Name = Barbuda
- - - - - Island Name = St. Kitts
- - - - - Island Name = Nevis
- - - - - Island Name = Anguilla

- - - - Windward Islands
- - - - - Grenada
- - - - - -  IslandName = Carriacou; Petite Martinique
- - - - - Island Name = Martinique
- - - - - Island Name = Barbados
- - - - - Island Name = St. Lucia
- - - - - Island Name = St. Vincent and the Grenadines

- - - Greater Antilles
- - - - Cuba
- - - - - Cayo Las Brujas
- - - - - - Canarreos Archipelago
- - - - - - - Island Name = Cayo Largo del Sur; Ernst Thälmann Island; Isla de la Juventud
- - - - - - Colorados Archipelago
- - - - - - - - Island Name = Banco Sancho Pardo; Cayo Alacranes; Cayo Buenavista; Cayo Ines de Soto; Cayo Jutías; Cayo Levisa; Cayo Punta Arenas; Cayo Rapado Grande; Cayo Santa María; Cayos Punta Tabaco
- - - - - - Jardines de la Reina
- - - - - - - - Island Name = Cayo Alcatraz; Cayo Algodon Grande; Cayo Anclitas; Cayo Breton; Cayo Caballones; Cayo Cabeza Del Este; Cayo Caguamas; Cayo De Las Doce Leguas; Cayos Ana Maria; Cayos Cinco Balas; Cayos Granada; Cayos Media Luna; Cayos Mordazo; Cayos Pingues
- - - - - - Jardines del Rey
- - - - - - - - Island Name = Cayo Coco; Cayo Esquivel; Cayo Guajaba; Cayo Guillermo; Cayo Romano; Cayo Sabinal; Cayo Saetia
- - - - - - Sabana-Camagüey Archipelago
- - - - - - - - Island Name =Cayo Fragoso
- - - - Island Name = Hispaniola
- - - - LocIsland Name =  Jamaica
- - - - Island Name = Puerto Rico

**Comment by rondlg** [comment]
# DECISION: Political hierarchy: **West Indies**

## PD0 [Continent] = South America

**Sovereign Nations** 
- PD1 [Country] = Trinidad and Tobago

## PD0 [Continent] = North America

**Sovereign Nations** 
- PD1 [Country] = Antigua and Barbuda	
- PD1 [Country] = Haiti
- PD1 [Country] = Bahamas	
- PD1 [Country] = Jamaica
- PD1 [Country] = Barbados	
- PD1 [Country] = Saint Lucia
- PD1 [Country] = Cuba	
- - PD2 [Province] 
- - - Pinar del Río; Artemisa; La Habana; Mayabeque; Matanzas; Cienfuegos; Villa Clara; Sancti Spíritus; Ciego de Ávila; Camagüey; Las Tunas; Granma; Holguín; Santiago de Cuba; Guantánamo
- - PD2 [Special Municipality] 
- - - Isla de la Juventud 
- PD1 [Country] = Saint Kitts and Nevis
- PD1 [Country] = Dominica	
- PD1 [Country] = Saint Vincent and the Grenadines
- PD1 [Country] = Dominican Republic	
- PD1 [Country] = Grenada

**Dependant Territories**
Sovereignty = United Kingdom
- PD1 [Country] = Anguilla; 
- PD1 [Country] = Virgin Islands (British)
- PD1 [Country] = Cayman Islands
- PD1 [Country] = Montserrat
- PD1 [Country] = Turks and Caicos Islands
- PD1 [Country] = Aguilla

Sovereignty = Netherlands
- PD1 [Country] = Aruba
- PD1 [Country] =  Curaçao
- PD1 [Country] =  Sint Maarten
- PD1 [Country] =  Bonaire
- PD1 [Country] =  Saba
- PD1 [Country] = Sint Eustatius

Sovereignty = United States
- PD1 [Country] = Puerto Rico

Sovereignty = Netherlands
- PD1 [Country] = Saint Barthélemy
- PD1 [Country] = Collectivity of Saint Martin
- PD1 [Country] = Guadeloupe
- PD1 [Country] = Martinique

Sovereignty = Venezuela
- PD1 [Country] = Nueva Esparta
- PD1 [Country] = Federal Dependencies of Venezuela

**Comment by rondlg** [comment]
See also [DECISON - Where to put Caribbean](https://github.com/fieldmuseum/EMu-Documentation/issues/28)

**Comment by cmaves3** [comment]
@rondlg  Addition of Antigua and Barbuda's PD2s. IRNs and additional info found [on this spreadsheet](https://docs.google.com/spreadsheets/d/1pG79ABtDlnhAATnMnhgbY2pJqZUsDPNZ4frforPP59E/edit?gid=461928516#gid=461928516).

Pol PD1 [Country]: Antigua and Barbuda
- PolPD2 [Dependancy]: Barbuda
- PolPD2 [Parish]: Saint George; Saint John; Saint Mary; Saint Paul; Saint Peter; Saint Philip
- - PolPD3 [Dependancy]; Redonda (PD2=Saint John Parish)

**Comment by cmaves3** [comment]
@rondlg Updates to correct and consolidate records of West Indies Island Group Hierarchy 

Greater Antilles: Merge 629287, 630926, 631220, and 631222. Assign 630759 as Parent
Lesser Antilles: Merge 631223 into 630990
Leeward Antilles: Create record with 630990	as Parent and Rank as Sub Group
Leeward Islands: For IRN 630989, change Parent to 630990 and change Rank to Sub Group
Windward Islands: Merge 631227 into 631226 and change Rank to Sub Group

Note: the above decision does not make it clear what rank Leeward Antilles/Islands and Windward Island should be. It is my suggestion that they be Sub Group.

**Comment by cmaves3** [comment]
@rondlg Addition of PD2/3 for several countries. IRNs and additional info found [on this spreadsheet](https://docs.google.com/spreadsheets/d/1pG79ABtDlnhAATnMnhgbY2pJqZUsDPNZ4frforPP59E/edit?gid=1318904507#gid=1318904507).

PolPD1 [Country]: Barbados
- PolPD2 [Parish]: Christ Church, Saint Andrew, Saint George, Saint James, Saint John, Saint Joseph, Saint Lucy, Saint Michael, Saint Peter, Saint Philip, Saint Thomas

PolPD1 [Country]: Belize
- PolPD2 [District]: Belize, Cayo, Corozal, Orange Walk, Stann Creek, Toledo

PolPD1 [Country]: Cuba
- PolPD2 [Province]: Artemisa, Camagüey, Ciego de Ávila, Cienfuegos, Granma, Guantánamo, Holguín, La Habana, Las Tunas, Matanzas, Mayabeque, Pinar del Río, Sancti Spíritus, Santiago de Cuba, Villa Clara
- PolPD2 [Special Municipality]: Isla de la Juventud

PolPD1 [Country]: Dominica
- PolPD2 [Parish]: Saint Andrew, Saint David, Saint George, Saint John, Saint Joseph, Saint Luke, Saint Mark, Saint Patrick, Saint Paul, Saint Peter

PolPD1 [Country]: Grenada
- PolPD2 [Parish]: Saint Andrew, Saint David, Saint George, Saint John, Saint Mark, Saint Patrick
- PolPD2 [Dependancy]: Southern Grenadine Islands

PolPD1 [Country]: Guyana
- PolPD2 [Region]: Barima-Waini, Cuyuni-Mazaruni, Demerara-Mahaica, East Berbice-Corentyne, Essequibo Islands-West Demerara, Mahaica-Berbice, Pomeroon-Supenaam, Potaro-Siparuni, Upper Demerara-Berbice, Upper Takutu-Upper Essequibo

PolPD1 [Country]: Haiti
- PolPD2 [Department]: Artibonite, Centre, Grandans, Grande’Anse, Latibonit, Lwès, Nip, Nippes, Nord, Nord-Est, Nord-Ouest, Nò, Nòdwès, Nòdès, Ouest, Sant, Sid, Sidès, Sud, Sud-Est

PolPD1 [Country]: Jamaica
- PolPD2 [Parish]: Clarendon, Hanover, Kingston, Manchester, Portland, Saint Andrew, Saint Ann, Saint Catherine, Saint Elizabeth, Saint James, Saint Mary, Saint Thomas, Trelawny, Westmoreland

PolPD1 [Country]: Saint Kitts and Nevis
- PolPD2 [State]: Nevis
- - PolPD3 [Parish]: Saint George Gingerland, Saint James Windward, Saint Paul Charlestown, Saint John Figtree, Saint Thomas Lowland
- PolPD2 [State]: Saint Kitts
- - PolPD3[Parish]: Christ Church Nichola Town, Saint Anne Sandy Point, Saint George Basseterre, Saint John Capisterre, Saint Mary Cayon, Saint Paul Capisterre, Saint Peter Basseterre, Saint Thomas Middle Island, Trinity Palmetto Point

PolPD1 [Country]: Saint Lucia
- PolPD2 [District]: Anse la Raye, Canaries, Castries, Choiseul, Dennery, Gros Islet, Laborie, Micoud, Soufrière, Vieux Fort

PolPD1 [Country]: Saint Vincent and the Grenadines
- PolPD2 [Parish]: Grenadines, Charlotte, Saint Andrew, Saint David, Saint George, Saint Patrick

PolPD1 [Country]: Trinidad and Tobago
- PolPD2 [City]: Port of Spain. San Fernando
- PolPD2 [Ward]: Tobago
- PolPD2 [Borough]: Arima, Chaguanas, Diego Martin, Point Fortin, Siparia
- PolPD2 [Region]: Couva-Tabaquite-Talparo, Mayaro-Rio Claro, Penal-Debe, Princes Town, San Juan-Laventille, Sangre Grande, Tunapuna-Piarco

**Comment by rondlg** [comment]
@cmaves3 - I have consolidated/cleaned up the West Indies Island Group Hierarchy island-group/sub-group records as requested:

- Greater Antilles: MASTER RECORD 630926
- Lesser Antilles: MASTER RECORD 630990
- Leeward Antilles: MASTER RECORD 630990
- Leeward Islands: MASTER RECORD 630989
- Windward Islands: MASTER RECORD 631226

Let me know if that all looks good

**Comment by cmaves3** [comment]
@rondlg I don't see a record for Leeward Antilles - that would be a new record, don't know if I should enter that. Other than that, all good!

**Comment by rondlg** [comment]
@cmaves3 New records are fair game, so go ahead and add that.

**Comment by cmaves3** [comment]
PolPD1 [Country]: Bahamas

- PolPD2 [District]: Acklins, Berry Islands, Bimini, Black Point, Cat Island, Central Abaco, Central Andros, Central Eleuthera, City of Freeport, Crooked Island and Long Cay, East Grand Bahama, Exuma, Grand Cay, Harbour Island, Hope Town, Inagua, Long Island, Mangrove Cay, Mayaguana, Moore's Island, New Providence, North Abaco, North Andros, North Eleuthera, Ragged Island, Rum Cay, San Salvador, South Abaco, South Andros, South Eleuthera, Spanish Wells, West Grand Bahama

PolPD1 [Country]: Dominican Republic

- PolPD2 [Province]: Azua, Baoruco, Barahona, Dajabón, Duarte, El Seibo, Elías Piña, Espaillat, Hato Mayor, Hermanas Mirabal, Independencia, La Altagracia, La Romana, La Vega, María Trinidad Sánchez, Monseñor Nouel, Monte Cristi, Monte Plata, Pedernales, Peravia, Puerto Plata, Samaná, San Cristóbal, San José de Ocoa, San Juan, San Pedro de Macorís, Sánchez Ramírez, Santiago, Santiago Rodríguez, Santo Domingo, Valverde
- PolPD2 [District]: Distrito Nacional

**Comment by cmaves3** [comment]
@rondlg Both the original and updated decision here have an apparent typo of repeating Netherlands as the Sovereignty of Saint Barthélemy, Saint Martin, Guadeloupe, and Martinique. France should be listed in Sovereignty for those lands.


> Sovereignty = Netherlands
> 
>     * PD1 [Country] = Aruba
> 
>     * PD1 [Country] =  Curaçao
> 
>     * PD1 [Country] =  Sint Maarten
> 
>     * PD1 [Country] =  Bonaire
> 
>     * PD1 [Country] =  Saba
> 
>     * PD1 [Country] = Sint Eustatius
> 
> 
> Sovereignty = United States
> 
>     * PD1 [Country] = Puerto Rico
> 
> 
> Sovereignty = Netherlands
> 
>     * PD1 [Country] = Saint Barthélemy
> 
>     * PD1 [Country] = Collectivity of Saint Martin
> 
>     * PD1 [Country] = Guadeloupe
> 
>     * PD1 [Country] = Martinique

**Comment by rondlg** [comment]
Converted to a discussion, to keep track of the agreed hierarchy
